appserver/web/gf-web-connector/src/main/java/fish/payara/appserver/context/ContextImpl.java

```diff
@@ -68,6 +68,19 @@ public void close() {
         private final InvocationManager invMgr;
     }
 
+    @RequiredArgsConstructor
```

Comment:
Remove lombok

Reply:
Copyright updated.

Once again, you can't just "remove lombok"  I will create a massive PR that replaces lombok with boilerplate and then an informed decision can be made.
For now, Lombok stays